Common/Tests/Insights/HyperwalletInsightsTests.swift

```diff
@@ -36,6 +36,7 @@ class HyperwalletInsightsTests: XCTestCase {
         if self.name.contains("testTrackError_ConfigNotInitialized") {
             return
         }
+        HyperwalletUI.clearInstance()
```

Comment:
Add separate func for clearInstance as suggested in Core SDK PR

Reply:
Added sperate testClearInstance